src/utilityModule.js

```diff
@@ -95,29 +95,34 @@ function getValidateMessage(requestBody) {
     throw new Error('Cannot find message data in request body');
   }
 
-  const message = new Message(JSON.stringify(data));
-  if(requestBody.properties && typeof requestBody.properties === 'object') {
-    const properties = requestBody.properties;
-    for(const property in properties) {
-      if (properties.hasOwnProperty(property)) {
-        message.properties.add(property, properties[property]);
+  try {
+    const message = new Message(JSON.stringify(data));
+    if(requestBody.properties && typeof requestBody.properties === 'object') {
+      const properties = requestBody.properties;
+      for(const property in properties) {
+        if (properties.hasOwnProperty(property)) {
+          message.properties.add(property, properties[property]);
+        }
       }
     }
-  }
 
-  if (requestBody.messageId) {
-    message.messageId = requestBody.messageId;
-  }
+    if (requestBody.messageId) {
+      message.messageId = requestBody.messageId;
+    }
 
-  if (requestBody.correlationId) {
-    message.correlationId = requestBody.correlationId;
-  }
+    if (requestBody.correlationId) {
+      message.correlationId = requestBody.correlationId;
+    }
 
-  if (requestBody.userId) {
-    message.userId = requestBody.userId;
-  }
+    if (requestBody.userId) {
+      message.userId = requestBody.userId;
+    }
 
-  return {channel, message};
+    return {channel, message};
+  }
+  catch(error) {
```

Comment:
>catch [](start = 2, length = 5)

this error message is focusing on the stringify failure so I think the catch should be at l.100 instead.